Playmate/Views/FriendRequestCell.m

```diff
@@ -56,14 +56,9 @@ - (void)setRequestInfo:(FriendRequest *)requestInfo {
     self.acceptButton.layer.cornerRadius = [Constants smallButtonCornerRadius];
     self.denyButton.layer.cornerRadius = [Constants smallButtonCornerRadius];
     
-    if (self.requester[@"profileImage"] != nil) {
-        UIImage* img = [UIImage imageWithData:[self.requester[@"profileImage"] getData]];
-        [self.profileImageView setImage:[Constants resizeImage:img withDimension:83]];
-    }
-    else {
-        UIImage* img = [UIImage imageNamed:@"playmate_logo_transparent.png"];
-        [self.profileImageView setImage:[Constants resizeImage:img withDimension:83]];
-    }
+    const BOOL hasProfileImage = (self.requester[@"profileImage"] != nil);
+    UIImage *img = hasProfileImage ? [UIImage imageWithData:[self.requester[@"profileImage"] getData]] : [Constants profileImagePlaceholder];
```

Comment:
we repeat this code in FriendCell.m and OutgoingRequestCell.m
Should we have a helper function to share code?